### DISCUSSION TITLE
assign fixed price to project

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#4223] [General]
I would like to be able to assign a lump-sum budget to a project so that this amount will be charged (no matter if a single hour or hundreds of hours were worked). When creating a project, I am aware that a budget can be defined (but the amount defined there is not charged in the invoice). I am further aware that "Prices" can be defined, either as hourly price, internal price or fixed price. However, in the latter case each record gets the same fixed price. 

The only (and quite relevant) situation where a fixed budget with the customer is agreed upon cannot be setup when creating the project (at least I could not find it). So, maybe a second switch could be added where the fixed price switch like this:

![image](https://github.com/kimai/kimai/assets/51003649/7de041e4-ae35-4a5a-a572-e54a469f201b)

**Comment by coleshirley** [comment]
+1 to this feature but this workaround does exist: https://github.com/kimai/kimai/discussions/3386
